app/townsquare/templates/townsquare/index.html

```diff
@@ -29,11 +29,50 @@
     <link rel="stylesheet" href="{% static "v2/css/tag.css" %}">
     <link rel="stylesheet" href="{% static "v2/css/theme_switcher.css" %}">
     <style>
+      .dark-mode img.hackathon-card-logo {
+        background: #fff;
+      }
+
+      .dark-mode time {
+        color: var(--gc-medium-dark-gray);
+      }
+
+      .dark-mode .hackathon-card-text {
+        color: var(--gc-medium-dark-gray);
+      }
+
+      .dark-mode .hackathon-card-text {
+        color: #fff;
+      }
+
+      .dark-mode .hackathon-card-text {
+        text-decoration: none;
+        color: #fff;
+      }
+
+      .dark-mode .box-hackathon {
+        background: #373350;
+      }
+
+      .dark-mode .box-hackathon a:hover {
+        text-decoration: none;
+        color: #fff;
+      }
+
+      .dark-mode .townsquare_block-header.box-collapse {
+        background: #373350;
+        border-bottom: 1px solid var(--gc-medium-dark-gray);
+      }
+
+      .hackathon-card-text {
+        color: #000;
+      }
```

Comment:
the darkmode css classes aren't needed! I think we can do without them
something like https://github.com/gitcoinco/web/pull/7062

Reply:
ok Thanks @thelostone-mc!